doc/user/load_save.md

```diff
@@ -523,6 +523,77 @@ outImages.push_back("bandicoot-favicon-inv.jpeg");
 mlpack::data::Save(outImages, matrix, info);
 ```
 
+### Resize images
+
+It is possible to resize images in mlpack with the following function:
+
+- `ResizeImages(images, info, newWidth, newHeight)`
+   * `images` an armadillo matrix contains a set of images, each one is
+     represeted in one column representing the image as a flattened vector.
+
+   * `info` will be populated with information from the image in `image`.
+
+   * `newWidth` the new width of the resized image.
+
+   * `newHeight` the new height of the resized image.
+
+   * This function does not return but it modifies the info object and images
+     matrix.
+
+   * ***NOTE:*** if the element type of the matrix is not `unsigned char` or
+   `float` (e.g. if `image` is not `arma::Mat<unsigned char>` or `arma::fmat`),
+   the matrix will be temporarily converted during resizing; therefore, using
+   `unsigned char` or `float` as the element type is the most efficient.
+
+   * This function expects all the images at the start to have identical
+     dimensions. If the images have different dimension, please assign each
+     image to one armadillo column and loop over all of them using a `for`
+     loop.
+    
+Example usage of the ResizeImages function on a set of images:
+
+```c++
```

Comment:
I'm not sure if the documentation snippets job will build right for this example.  I know that all the other example code doesn't use the `mlpack` namespace so that it can be trivially pasted, so at least it will probably need a bunch of `mlpack::`s in it.  You can test the compilation locally with this command:

```
CXX=g++ scripts/test-docs.sh doc/user/load_save.md
```

from the root of the repository.

It's awesome that you have both usages (single image and multiple images) in here, but maybe we should have them as separate examples?  I just mean breaking the code block after "Now all the images have the same dimension", adding a little line that says it is an example about resizing images in batch, and then a new code block.